app/components/preprint-form-authors.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
```

Comment:
So what I'm hearing is that there are major drawbacks to this that effect code, not to mention the continuous confusion of looking at a lot of component.js tabs during development while the benefits are 1 - consistency, we are already doing it elsewhere, 2 - we might forget to remove files when components are no longer needed, 3 - ember is moving towards something like pods. 

I'm genuinely curious about what benefits for code quality makes using pods worthwhile to have roundabout ways to solve issues like recognizing styles outside styles/ folder for instance. Sam might have some more insights on this. I'll ask him to join in.

I think the project scope is small enough, and components few enough that we can restructure this later.